tests/backstop/bin/backstop

```diff
@@ -20,15 +20,21 @@ ROOT_DIR="$(dirname "$BIN_DIR")/../../"
 VERSION=6.1.1
 IMAGE=googlesitekit/vrt:$VERSION
 
-# (re)Build the backstopjs image as needed.
-docker build \
-	--pull \
-	--tag $IMAGE \
-	--build-arg BACKSTOPJS_VERSION=$VERSION \
-	"$ROOT_DIR/tests/backstop"
+# Determine which containerisation tool to use. If Docker is available,
+# use it; otherwise, use Podman.
+command -v docker > /dev/null && DC="docker" || DC="podman"
```

Comment:
This isn't really necessary as Podman can be installed as an alias for `docker` (or symlink). I think it's better to use `docker` as the interface which is what most people have/use anyways.

Reply:
Ok, i'll revert it shortly.